### PR TITLE
added scripts for terminal blocks

### PR DESCRIPTION
**Pull request by jkriege2**
@pointhi ... can you merge this?

used for https://github.com/KiCad/TerminalBlock_RND.pretty/pull/1
https://github.com/KiCad/TerminalBlock_MetzConnect.pretty/pull/1

### CODE REVIEW

**Comment by jkriege2** [comment]
please keep this open for now, as I'm going to extend this a bit more to cover most of the terminals that we have in the lib!

Thanks,
JAN

File: scripts/tools/drawing_tools.py [hunk 1]
```diff
@@ -346,6 +346,11 @@ def addVDLineWithKeepout(kicad_mod, x, y0, y1, layer, width, keepouts=[], roun=0
 
 
 
+# split a rectangle 
+def addRectWith(kicad_mod, x, y, w, h, layer, width, roun=0.001):
+	kicad_modg.append(RectLine(start=[roundG(x, roun),roundG(y, roun)], end=[roundG(x+w, roun),roundG(y+h, roun)], layer=layer, width=width))
```
Comment (pointhi) [inline scripts/tools/drawing_tools.py]: at least replace this tab with spaces

Comment (jkriege2) [reply scripts/tools/drawing_tools.py]: should be fixed now

**Comment by jkriege2** [comment]
added ... scripts for complete rework of all terminal blocks:

2017-11-11 18_23_20-pcbnew d__kicad_kicad_review-project2_kicad_review-project2 kicad_pcb


... MORE TO COME, PLEASE DON'T MERGE YET!

**Comment by pointhi** [comment]
Thanks for this contribution

**Comment by jkriege2** [comment]
@pointhi  Sorry, it took me a few days to notice ... the conflicts are fixed now!

Best,
JAN

**Comment by pointhi** [comment]
thanks for fixing

**Pull request merged**